package.json

```diff
@@ -16,7 +16,6 @@
     "react-bs-notifier": "^4.4.6",
     "react-copy-to-clipboard": "^5.0.1",
     "react-dom": "^16.4.1",
-    "react-dropzone": "^4.2.11",
```

Comment:
Did you made a test in order to know if `react-dropzone` is used somewhere else before removing it?